pkg/asset/machines/worker.go

```diff
@@ -295,7 +295,10 @@ func (w *Worker) Generate(dependencies asset.Parents) error {
 			}
 		case azuretypes.Name:
 			mpool := defaultAzureMachinePoolPlatform()
-			mpool.InstanceType = azuredefaults.ComputeInstanceType(installConfig.Config.Platform.Azure.Region)
+			mpool.InstanceType = azuredefaults.ControlPlaneInstanceType(
```

Comment:
Should this be using `ComputeInstanceType`?

Reply:
yes will fix it now.